stripe/src/main/java/com/stripe/android/paymentsheet/viewmodels/SheetViewModel.kt

```diff
@@ -53,6 +53,11 @@ internal abstract class SheetViewModel<TransitionTargetType>(
     private val _transition = MutableLiveData<TransitionTargetType?>(null)
     internal val transition: LiveData<TransitionTargetType?> = _transition
 
+    /**
+     * On BaseAddCard this is set every time the add card is determined to be valid
```

Comment:
Thanks for adding comments. You can make the class names linkable by surrounding them with brackets (e.g. `[BaseAddCard]`)